test/performance/odp_sched_perf_run.sh

```diff
@@ -35,6 +38,18 @@ run()
 			echo odp_sched_perf FAILED
 			exit $RET_VAL
 		fi
+
+		echo odp_sched_perf -p 2 -c $1
+		echo ===============================================
+		$TEST_DIR/odp_sched_perf${EXEEXT} -p 2 -c $1
+		RET_VAL=$?
+		if [ $RET_VAL -eq 77]; then
```

Comment:
Missing whitespace before closing square bracket causes a warning: `./odp_sched_perf_run.sh: 46: [: missing ]`

Also, `TEST_SKIPPED` variable could be used in the comparison.